IHP/IDE/SchemaDesigner/Controller/Columns.hs

```diff
@@ -228,3 +228,9 @@ isCreateEnumType CreateEnumType {} = True
 isCreateEnumType _ = False
 
 nameList statements = map (get #name) statements
+
+arrayifytype :: Bool -> PostgresType -> PostgresType
+arrayifytype False   (PArray coltype) = coltype
+arrayifytype True  a@(PArray coltype) = a
```

Comment:
Is this an expected case? I feel like this should never happen. (if so, an error of sorts might be better)

Reply:
Yes. This is when editing. It could be an array type or not before edit, and the checkbox could turn it on or off. So all cases are viable.